app/kumactl/pkg/config/validate.go

```diff
@@ -1,46 +1,28 @@
 package config
 
 import (
-	"encoding/json"
-	"io"
-	"net/http"
+	"context"
 	"time"
 
 	"github.com/pkg/errors"
 
-	"github.com/kumahq/kuma/pkg/api-server/types"
+	"github.com/kumahq/kuma/app/kumactl/pkg/client"
+	kumactl_resources "github.com/kumahq/kuma/app/kumactl/pkg/resources"
 	kumactl_config "github.com/kumahq/kuma/pkg/config/app/kumactl/v1alpha1"
 	"github.com/kumahq/kuma/pkg/version"
 )
 
 func ValidateCpCoordinates(cp *kumactl_config.ControlPlane, timeout time.Duration) error {
-	req, err := http.NewRequest("GET", cp.Coordinates.ApiServer.Url, nil)
```

Comment:
Looks like some problem with TLS configuration `could not configure HTTP client with TLS`